src/steps/02-EvaluationCriteria/JandA/DescriptionOfJustification.vue

```diff
@@ -6,20 +6,155 @@
           <h1 class="mb-3">
             Let’s work on your justification for this exception to fair opportunity
           </h1>
-          <p>
-            Intro paragraph
+          <p v-if="selectedException === 'YES_FAR_16_505_B_2_I_B'">
+            Briefly describe how this action requires use of the “unique or highly
+            specialized capabilities” exception. Discuss the events, history, and
+            circumstances that led to the current situation requiring use of procedures
+            for other than fair opportunity competition. Your rationale should include
+            a discussion of CSP name's unique qualifications for fulfilling the order
+            requirements.
           </p>
+          <p v-else-if="selectedException === 'YES_FAR_16_505_B_2_I_C'">
+            Briefly describe how this action requires use of the “logical follow-on”
+            exception. Discuss the events, history, and circumstances that led to the
+            current situation requiring use of procedures for other than fair opportunity
+            competition. Your rationale should describe why the relationship between the
+            initial order issued under JWCC and the follow-on is logical (e.g., in terms
+            of scope, period of performance, or value).
+          </p>
+          <div v-else-if="selectedException === 'YES_FAR_16_505_B_2_I_A'">
+            <p>
+              Briefly describe how this action requires use of the “unusual and compelling
+              urgency” exception. Discuss the events, history, and circumstances that led
+              to the current situation requiring use of procedures for other than fair
+              opportunity competition. Your rationale should document the required delivery
+              schedule and lead-times involved, including a chronological explanation of events
+              that caused the exigent situation.
+            </p>
+            <ATATExpandableLink aria-id="ContactFAQ2">
+              <template v-slot:header>
+                What else should I include in my description of the justification?
+              </template>
+              <template v-slot:content>
+                <p>
+                  Be sure to explain any actual or apparent time lags between events.
+                  At the very least, include:
+                </p>
+                <ul>
```

Comment:
Spacing is significantly off here (left is browser, right is Figma). Please add class "mb-4" to the `p` tag, and add class `_atat-ul` to the `ul` tag - it's a helper class we use to ensure correct list line spacing
<img width="488" alt="Screen Shot 2023-04-24 at 7 53 19 PM" src="https://user-images.githubusercontent.com/90333349/234139632-2dfa6e3f-4bb6-41f8-a9cc-fd73bcaf93a7.png">